### ISSUE TITLE
potential build issue

### ISSUE THREAD

**Issue by terrenewebsolutions** [SongProOrg/songpro-javascript#3]
Hi, when installing this package from npm, it appears that only the license, readme, and package.json files are included in the node_modules folder.  I checked out the project and attempted to build it, but got the following error about the babel version 

```
ERROR in ./src/SongPro.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error: Requires Babel "^7.0.0-0", but was loaded with "6.26.3". If you are sure you have a compatible version of @babel/core, it is likely that something in your build process is loading the wrong version. Inspect the stack trace of this error to look for the first entry that doesn't mention "@babel/core" or "babel-core" to see what is calling Babel. (While processing preset: "/home/chris/songpro-js/node_modules/@babel/preset-env/lib/index.js")
    at throwVersionError (/home/chris/songpro-js/node_modules/@babel/helper-plugin-utils/lib/index.js:65:11)
    at Object.assertVersion (/home/chris/songpro-js/node_modules/@babel/helper-plugin-utils/lib/index.js:13:11)
    at /home/chris/songpro-js/node_modules/@babel/preset-env/lib/index.js:188:7
    at /home/chris/songpro-js/node_modules/@babel/helper-plugin-utils/lib/index.js:19:12
    at /home/chris/songpro-js/node_modules/babel-core/lib/transformation/file/options/option-manager.js:317:46
    at Array.map (<anonymous>)
    at OptionManager.resolvePresets (/home/chris/songpro-js/node_modules/babel-core/lib/transformation/file/options/option-manager.js:275:20)
    at OptionManager.mergePresets (/home/chris/songpro-js/node_modules/babel-core/lib/transformation/file/options/option-manager.js:264:10)
    at OptionManager.mergeOptions (/home/chris/songpro-js/node_modules/babel-core/lib/transformation/file/options/option-manager.js:249:14)
    at OptionManager.init (/home/chris/songpro-js/node_modules/babel-core/lib/transformation/file/options/option-manager.js:368:12)
    at File.initOptions (/home/chris/songpro-js/node_modules/babel-core/lib/transformation/file/index.js:212:65)
    at new File (/home/chris/songpro-js/node_modules/babel-core/lib/transformation/file/index.js:135:24)
    at Pipeline.transform (/home/chris/songpro-js/node_modules/babel-core/lib/transformation/pipeline.js:46:16)
    at transpile (/home/chris/songpro-js/node_modules/babel-loader/lib/index.js:50:20)
    at Object.module.exports (/home/chris/songpro-js/node_modules/babel-loader/lib/index.js:173:20)
```
is it possible the latest build is broken?

**Comment by spilth** [comment]
Hi @terrenewebsolutions. I'm finally taking a look at this and should have a fix up soon.

**Comment by spilth** [comment]
Fixed by #35
